core/math/bvh_split.inc

```diff
@@ -20,8 +20,8 @@ void _split_leaf_sort_groups_simple(int &num_a, int &num_b, uint16_t *group_a, u
 		group_b[num_b++] = ind;
 
 		// remove from a
-		group_a[0] = group_a[num_a - 1];
 		num_a--;
+		group_a[0] = group_a[num_a];
 		return;
```

Comment:
Would expect optimizer to re-order this anyway (unless there was some conversion going on for the index), but the change is fine. :+1: